### ISSUE TITLE
Decide on a process for editing this specification

### ISSUE THREAD

**Issue by RubenVerborgh** [solid/specification#1]

**Comment by Mitzi-Laszlo** [comment]
https://github.com/solid/information/blob/master/decision-making-processes.md

**Comment by Mitzi-Laszlo** [comment]
https://github.com/solid/information/pull/138

**Comment by RubenVerborgh** [comment]
Suggestion from my side: I would use https://github.com/solid/information/blob/master/decision-making-processes.md to make a decision on what _process_ to use for spec writing, but not adopt that process directly as the spec writing process itself.

From experience, having 2—3 dedicated editors on a spec is a good mechanism. (To be decided then: PRs, authors, etc.)

**Comment by Mitzi-Laszlo** [comment]
Ok, sounds good although needs more detail. For example, how are the editors appointed? What are their responsibilities? How would the editors resolve differences of opinion that inevitably arise between them?

**Comment by RubenVerborgh** [comment]
> For example, how are the editors appointed?

Some people step up, usually it's a mutual agreement and a group forms. It's rare that elections have to be held 🙂

> What are their responsibilities?

Delivering a clear, consistent, readable, implementable spec.

> How would the editors resolve differences of opinion that inevitably arise between them?

That is rarer than you might think. Editors generally just facilitate the translation of a consensus into a text.

When there would be difference, we discuss with the group, and a common solution is derived.

> The reason I'm asking these questions is that the editors need respected authority to be able to carry out their work.

There's less authority than we might assume. They are not deciders; they are people who write down what all/most of us want. The community needs to trust them as skilled and dedicated writers, not necessarily as leaders.

> It needs to be clear what that work is and where and on what external individuals can comment and how to expect those comments to be handled.

Editors are the ones moving input to the text; but the inputs themselves can be given by anyone, and anyone can comment on them. Editors usually don't decide on yes or no; they decide when a topic has been sufficiently discussed and then move it to the text. When there is insufficient discussion, they moderate.

To summarize: editors facilitate technical discussion and translate the resulting consensus into a text, that's it.

**Comment by Mitzi-Laszlo** [comment]
Here is a [proposal](https://github.com/solid/information/pull/138) for the creation of the Solid spec v1  which would need to be approved by the Solid Leader.

**Comment by RubenVerborgh** [comment]
Deferred to https://github.com/solid/culture/pull/6 and its follow-up.
